clients/js/package.json

```diff
@@ -2,7 +2,7 @@
   "type": "module",
   "name": "@oasisprotocol/sapphire-paratime",
   "license": "Apache-2.0",
-  "version": "1.2.0",
+  "version": "1.2.2",
```

Comment:
Not `1.2.1`?

Reply:
I attempted to release 1.2.1 via a tag but it failed.
Unable to remove tags or force push tags, so using 1.2.2